inspirehep/modules/records/exceptions.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+#
```

Comment:
usually we use ``errors.py`` to be consistent with invenio :)

Reply:
Except my exceptions.py in records there are only two files with Exceptions, one errors.py for Workflows and one exceptions.py for orcid. So for current code we can say that `usually` we use exceptions.py ;) But to be honest I do not see any written information where we should declare exception types/classes.

Reply:
That's true we don't have any written information, I'll stick with the invenio naming just because we are extending its functionality especially in this case.

Reply:
You are right that as we are extending the functionality of Invenio we should with the invenio naming...
BUT actually I do not like to create exceptions in errors.py file as it's simple example of code obfuscation. (Who is looking for exceptions in file called errors?)
Also naming exceptions as errors looks like reducing the importance of it. We shouldn't repeat Invenio mistakes/problems IMHO.